spaces/views.py

```diff
@@ -1,3 +1,68 @@
-from django.shortcuts import render
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Min, Q
 
-# Create your views here.
+from spaces.models    import Space, District
+from orders.models    import OrderStatus
+
+class MenuPageView(View):
```

Comment:
```suggestion
class ProductsView(View):
```
저희는 페이지 단위가 아닌 자원 단위로 생각해야 합니다. 해당 API가 무조건 mainpage에서만 사용되는 것이 아니고, 다른 곳에서도 충분히 사용될 수 있습니다.